src/components/HoldMenuSectionList.js

```diff
@@ -0,0 +1,59 @@
+import _ from 'lodash';
+import React from 'react';
+import {View} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@styles/useThemeStyles';
+import variables from '@styles/variables';
+import Icon from './Icon';
+import * as Illustrations from './Icon/Illustrations';
+
+function HoldMenuSectionList() {
```

Comment:
I think you could create this component in Typescript already. Especially since it's quite a simple one 😄